course-v2/assets/js/mobile_course_drawers.ts

```diff
@@ -0,0 +1,48 @@
+enum SWIPE_DIRECTION {
```

Comment:
**Suggestion:** enums and their members are usually PascaleCase, so `SwipeDirection`.

But apparently #926 doesn't enforce this, hence just a suggestion.